v3/src/models/data/data-set.ts

```diff
@@ -132,12 +132,13 @@ export const DataSet = V2Model.named("DataSet").props({
   // ordered parent-most to child-most
   collections: types.array(CollectionModel),
   attributesMap: types.map(Attribute),
-  itemIds: types.array(types.string),
+  _itemIds: types.array(types.string),
   sourceName: types.maybe(types.string),
   description: types.maybe(types.string),
   importDate: types.maybe(types.string),
   // for serialization only, not for dynamic selection tracking
-  snapSelection: types.array(types.string)
+  snapSelection: types.array(types.string),
+  hiddenItems: types.array(types.string)
```

Comment:
Given that we have "hidden" in the data display world, it would be more consistent to use "setAside" here.

Reply:
I did consider `setAside` and decided that I prefer `hidden` in this case. The `DataSet` is used in other applications besides CODAP (e.g. CLUE) and is intended to have a data-set-centric API, not a CODAP-centric one. From the standpoint of the `DataSet`, those items/cases are hidden, and I don't think there's any inconsistency with CODAP calling them `setAside` for CODAP reasons, while other applications may call it something else.